tests/visual/scrollview.html

```diff
@@ -13,8 +13,8 @@
 <body>
 
     <div id="test-area" class="grid">
```

Comment:
Add an Angular `Scrollview` visual test with the current rendering. Notify with comment which one is Angular and which is jQuery markup.

Reply:
also, add a TODO comment in the visual test for Angular to be updated when we refactor the component's rendering